packages/docusaurus-theme-redoc/src/utils/useSpec.ts

```diff
@@ -16,10 +17,11 @@ let currentStore: AppStore | null = null;
  * (c) 2024 Rohit Gohri
  * Released under the MIT License
  */
-export function useSpec(
-  { spec, url, themeId }: SpecProps,
+export default function useSpec(
```

Comment:
Can we undo this? I prefer using named exports in my projects. And this makes it inconsistent with other files

Reply:
normaly, if your file has one function, you should export it as default. But I can undo it, you're the boss

Reply:
fixed